src/main/java/org/spongepowered/api/event/rcon/RConCommandEvent.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered.org <http://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.event.rcon;
+
+import org.spongepowered.api.event.message.CommandEvent;
+import org.spongepowered.api.util.command.source.RConSource;
+
+public interface RConCommandEvent extends CommandEvent, RConEvent {
+
+    @Override
+    RConSource getSource();
+
+    /**
+     * Gets whether {@link RConSource#getLoggedIn()} will be ignored when
+     * determining whether to execute the supplied {@link #getRawPayload() command}.
+     *
+     * <p>Normally, if an {@link RConSource} is not {@link RConSource#getLoggedIn()} logged in},
+     * the supplied {@link #getRawPayload() command} will not be executed.
+     *
+     * The opposite scenario, preventing the execution of a command by a logged in
+     * client, can be achieved with {@link #setCancelled(boolean)}.</p>
+     *
+     * @return Whether to override the client's logged in status
+     */
+    boolean overrideLoggedIn();
```

Comment:
I'd appreciate any suggestions for improving the name of this method and the corresponding setter.